packages/connect/src/api/evoluGetNode.ts

```diff
@@ -0,0 +1,27 @@
+import { Assert } from '@trezor/schema-utils';
+
+import { PROTO } from '../constants';
+import { AbstractMethod } from '../core/AbstractMethod';
+
+export default class EvoluGetNode extends AbstractMethod<'evoluGetNode', PROTO.EvoluGetNode> {
+    hasBundle?: boolean;
+
+    init() {
```

Comment:
These params should be set:
```suggestion
    init() {
        this.requiredPermissions = ['read'];
        this.firmwareRange = getFirmwareRange(this.name, null, this.firmwareRange);
```
And supported firmware range needs to be configured in `connect/src/data/config.ts`

Reply:
done